src/m1_facilitator/models/Transaction.ts

```diff
@@ -15,22 +15,34 @@
 import BigNumber from 'bignumber.js';
 import Comparable from '../../common/observer/Comparable';
 
+/**
+ * Type of statuses.
```

Comment:
Please add document, when these status are updated.
What is pending, sent, and failed

Reply:
Done!